### DISCUSSION TITLE
sql row limit 1000000

### DISCUSSION THREAD

**Discussion by snoof** [teslamate-org/teslamate#2749] [Q&A]
is there a way to remove from sql  the limit  1000000 ? 
i have the above problem .. 
![image](https://user-images.githubusercontent.com/34168204/179076795-d51b5849-92cc-43a6-bc6b-1ff6808064c2.png)

**Comment by JingxinLee** [comment]
same here

**Comment by Bzzz** [comment]
+1 (needed for a 17 day trip, 1M rows is roughly 13 days)
